### PR TITLE
381 paper submission

### PR DESCRIPTION
**Pull request by YipengHu**
# Description

prepare manuscript for submission

Fixes #<issue_number>

## Type of change

- [ ] Documentation Update (fix or improvement on the documentation)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR.
If you're unsure about any of them, don't hesitate to ask. We're here to help! This is
simply a reminder of what we are going to look for before merging your code._

- [ ] I have
      [installed pre-commit](https://deepreg.readthedocs.io/en/latest/contributing/setup.html)
      and formatted all changed files.
- [ ] My commits' message styles matches
      [our requested structure](https://deepreg.readthedocs.io/en/latest/contributing/commit.html),
      e.g. `Issue #<issue number>: detailed message`.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/joss_paper/paper.bib [hunk 1]
```diff
@@ -118,7 +118,7 @@ @article{vercauteren2009diffeomorphic
 @article{brown2020introduction,
```
Comment (NMontanaBrown) [inline docs/joss_paper/paper.bib:118]: Name not: (Brown, Nina Montana), replace with (Montana Brown, Nina).
Off the top of my head, these are names we have used prior: Shaheer U. Saeed (@s-sd) and Zachary M. C. Baum (@zacbaum ), check with auths.

File: docs/joss_paper/paper.md [hunk 2]
```diff
@@ -71,12 +71,10 @@ Image fusion is a fundamental task in medical image analysis and computer-assist
...
 
 # Algorithms
-In this paper, we first summarise several standard pairwise image registration network training strategies, in which the network aims to align a pair of moving- and fixed images such that the moving image can be warped or transformed into the fixed image coordinates. The methodologies adopted in these algorithms are building blocks of many other registration tasks, such as group-wise registration and morphological template construction [@dalca2019learning; @siebert2020deep; @luo2020mvmm].
+In this section, we first summarise several standard pairwise image registration network training strategies, in which the network aims to align a pair of moving- and fixed images such that the moving image can be warped or transformed into the fixed image coordinates. The methodologies adopted in these algorithms are building blocks of many other registration tasks, such as group-wise registration and morphological template construction [@dalca2019learning; @siebert2020deep; @luo2020mvmm].
```
Comment (NMontanaBrown) [inline docs/joss_paper/paper.md:77]: How about, "In this section we first summarise several standard pairwise image registration network training strategies  that are implemented in the DeepReg package."

**Review by NMontanaBrown — COMMENTED** [review]
Acknowledging the i4health CDT. I am not sure if they also support @zacbaum @QianyeYang and @RemiDelaunay or if they need to acknowledge this source of funding -> suggested.
"NMB, (ZB, QY? RD?) is (are) supported by the EPSRC CDT i4health (EP/S021930/1)"

**Comment by YipengHu** [comment]
> Acknowledging the i4health CDT. I am not sure if they also support @zacbaum @QianyeYang and @RemiDelaunay or if they need to acknowledge this source of funding -> suggested.
> "NMB, (ZB, QY? RD?) is (are) supported by the EPSRC CDT i4health (EP/S021930/1)"

NMB, ZB, QY RD are also supported by the EPSRC CDT i4health (EP/S021930/1).

File: docs/joss_paper/paper.bib (hunk 1 above)
Comment (s-sd) [reply docs/joss_paper/paper.bib:118]: Thank you! I think if a change is possible, 'Shaheer U. Saeed' might be better

**Review by DeepRegNet — DISMISSED** [review]

**Review by mathpluscode — APPROVED** [review]

**Pull request merged**